clinica/iotools/converters/genfi_to_bids/genfi_to_bids.py

```diff
@@ -0,0 +1,94 @@
+"""Convert the GENFI dataset into BIDS."""
+
+from os import PathLike
+from typing import Optional
+
+
+def convert_images(
+    path_to_dataset: PathLike,
+    bids_dir: PathLike,
+    path_to_clinical: Optional[PathLike],
+    gif: bool,
```

Comment:
Wouldn't it make more sense to have this as an optional parameter defaulting at True (or False maybe..) ?

Reply:
It is not optional for this function. It is for the command line, but this function will always receive a value for gif